docs/Manual/10 Packages.md

```diff
@@ -2,45 +2,45 @@
+The result has two files: `MyPackage.hat` and `MyPackage.lib`. The output directory defaults to current working directory. Though we can change the output directory with this:
 
 ```python
 package.build(format=acc.Package.Format.HAT_DYNAMIC, name="myPackage", platform=acc.Package.Platform.WINDOWS, output_dir="hat_packages")
 ```
 
 ## MLIR package format
-MLIR format is mainly used for debugging purposes, and as a way to follow the multiple lowering MLIR stages, from Accera DSL all the way to runnable code.
+MLIR format is primarily used for debugging purposes and for following the multiple lowering MLIR stages, from Accera DSL, all the way to runnable code.
 ```python
 package.build(format=acc.Package.Format.MLIR, name="myPackage")
 ```
 
 ## Function names in packages
-When a function is added to a package, the user specifies its base name. The full function name is the base name followed by an automatically generated unique identifier. For example, if the base name is "myFunc" then the function name could be "myFunc_8f24bef5". If no base name is given, the function name is just the automatically-generated unique identifier.
+We can specify the base name of a function when it is added to a package. The full function name is the base name followed by an automatically generated unique identifier. For example, if the base name is "myFunc" then the function name could be "myFunc_8f24bef5". If no base name is defined, the automatically-generated unique identifier becomes the function name.
 
-The unique identifier ensures that no two functions share the same name, but also makes it harder to call the function from client code. Specifically, each time that the Accera package is updated and rebuilt, the function name could change. Therefore, the HAT file also includes the client code to call the function without the unique identifier. Concretely, if the function signature in C is
+The unique identifier ensures that no two functions share the same name. However, invoking the function from the client code becomes cumbersome because the function name could change each time the Accera package is updated and rebuilt. Therefore, the HAT file includes the client code to call the function without the unique identifier. Concretely, if the function signature in C is:
 ```
 void myFunc_8f24bef5(const float* A, float* B);
 ```
 then the HAT file also contains the line:
 ```
 void (*myFunc)(const float* A, float* B) = myFunc_8f24bef5;
 ```
-The above basically makes the abbreviated name `myFunc` a synonym of the full function name `myFunc_8f24bef5`. If multiple functions share the same base name, an arbitrary one of them gets the abbreviation.
+The above code makes the abbreviated name `myFunc` an alias of the full function name `myFunc_8f24bef5`. If multiple functions share the same base name, one of them randomly gets the alias.
```

Comment:
"If multiple functions share the same base name, the first function in the HAT file gets the alias."

Reply:
I was wondering how this randomness work. Thanks for clarifying.